modules/users/server/routes/users.server.routes.js

```diff
@@ -26,6 +27,17 @@ module.exports = function (app) {
     .all(usersPolicy.isAllowed)
     .post(userProfile.avatarUploadField, userProfile.avatarUpload);
 
+  app
+    .route('/api/users/blocked-users')
+    .all(usersPolicy.isAllowed)
+    .get(userBlock.getBlockedUsers);
```

Comment:
What would be scenario where we need to fetch a list of IDs via API?

Reply:
I thought of using it on a screen for unblocking users. We list all blocked users (api return the list of blocked users populated with simplified profiles) and then the user can choose one to remove.

Reply:
Good thinking 👍  :-)